torch/_inductor/compile_fx.py

```diff
@@ -670,6 +671,9 @@ def fx_codegen_and_compile(
             payload_fn=lambda: gm.print_readable(print_output=False),
         )
         optimus_scuba_log["inductor"] = counters["inductor"]
+        runtime_env: RuntimeEnvironment = RuntimeEnvironment()
+        optimus_scuba_log["model_type"] = runtime_env.get_root_workflow_model_type_name()
+        optimus_scuba_log["global_rank"] = runtime_env.get_global_rank()
```

Comment:
ditto

Reply:
TIL https://github.com/pytorch/pytorch/blob/main/torch/_utils_internal.py is exposed to external.
but `RuntimeEnvironment` is still internal.